cfe_ts_crc.c

```diff
@@ -125,30 +127,43 @@ int main(int argc, char **argv)
     if (fd < 0)
     {
         printf("\ncfe_ts_crc error: can't open input file!\n");
-        exit(0);
+        printf("%s\n", strerror(errno));
+        exit(1);
```

Comment:
A common POSIX paradigm here is to just do something like this:

    if (fd < 0)
    {
         perror(argv[1]);
         exit(1);
    }

This prints the file name that the user specified (helpful to show in case it was the result of a shell glob or something - might not match what they had on command line) along with the error reported by the OS with a minimum of effort.

Reply:
fixed